plugins/source/gcp/CONTRIBUTING.md

```diff
@@ -2,15 +2,16 @@
 
 Thanks for contributing to CloudQuery! You are awesome. This document serves as a guide for adding new services and resources to the GCP source plugin.
 
-There are three main steps to adding a new GCP resource:
-1. [Add a code generation recipe](#1-add-a-code-generation-recipe)
-2. [Writing the resolver function to fetch the resource using the AWS SDK](#2-setting-up-the-resource)
+There are two main steps to adding a new GCP resource:
 
-## 1. Add a Code Generation Recipe
```

Comment:
Why remove the step numbers from the headings here? I think that helps to add context and ensures readers don't miss a step by skipping around

Reply:
Ah it feels odd to have numbering for a header. I can keep those I don't mind

Reply:
Probably not worth spending more time on, but I would also be okay with `Step 1: ...`, `Step 2: ...`, etc if that's better for you :)

Reply:
I don't have a strong opinion here so we can leave it at that :)